README.md

```diff
@@ -1,3 +1,135 @@
 # scats
-
 Selenium Cucumber automated testing system
+
```

Comment:
could we allow other projects to include scats-core jar file into lib dependency and use it? This is better for anyone to use the right version and to upgrade/ downgrade versions easily.